example/animated_pie_chart_example.dart

```diff
@@ -11,32 +11,32 @@ final List<List<CircularStackEntry>> _quarterlyProfitPieData = [
   <CircularStackEntry>[
     new CircularStackEntry(
       <CircularSegmentEntry>[
-        new CircularSegmentEntry(500.0, Colors.red[200], rankKey: 'Q1'),
-        new CircularSegmentEntry(1000.0, Colors.green[200], rankKey: 'Q2'),
-        new CircularSegmentEntry(2000.0, Colors.blue[200], rankKey: 'Q3'),
-        new CircularSegmentEntry(1000.0, Colors.yellow[200], rankKey: 'Q4'),
+        new CircularSegmentEntry(500.0, Colors.red[200]!, rankKey: 'Q1'),
+        new CircularSegmentEntry(1000.0, Colors.green[200]!, rankKey: 'Q2'),
+        new CircularSegmentEntry(2000.0, Colors.blue[200]!, rankKey: 'Q3'),
+        new CircularSegmentEntry(1000.0, Colors.yellow[200]!, rankKey: 'Q4'),
```

Comment:
this is all example code, so seems safe to force-unwarp these colors

Reply:
seems fine. love that this is using `new` still 😬